package.json

```diff
@@ -97,6 +97,7 @@
     "@elastic/apm-rum-react": "^1.3.1",
     "@elastic/charts": "34.2.1",
     "@elastic/datemath": "link:bazel-bin/packages/elastic-datemath",
+    "@elastic/apm-generator": "link:bazel-bin/packages/elastic-apm-generator",
```

Comment:
I see the notes say to call the script directly from the package. Is it to be used outside the package, if not, it can be removed from the `package.json`

Reply:
If not, it seems more like a `devDepenedency` which we shouldn't ship with the distribution.

Reply:
I would like to use this (the library, not the script) in the API tests. I'll move it to a devDependency.